tests/shared/src/test/scala/scala/meta/tests/parsers/dotty/InfixSuite.scala

```diff
@@ -256,14 +256,92 @@ class InfixSuite extends BaseDottySuite {
          |    | boiling
          |}
          |""".stripMargin,
-      Some(
-        """|{
-           |  freezing
-           |  |.boiling
-           |}
-           |""".stripMargin
-      )
-    )(Term.Block(List(tname("freezing"), Term.Select(tname("|"), tname("boiling")))))
+      """|{
+         |  freezing
+         |  |boiling
+         |}
+         |""".stripMargin
+    )(blk("freezing", tpostfix("|", "boiling")))
+  }
+
+  test("scala3 infix syntax 3.2.1") {
+    runTestAssert[Stat](
+      """|{
+         |  freezing
+         |
+         |    `|` boiling
+         |}
+         |""".stripMargin,
+      """|{
+         |  freezing
+         |  |boiling
+         |}
+         |""".stripMargin
+    )(blk("freezing", tpostfix("|", "boiling")))
+  }
+
+  test("scala3 infix syntax 3.2.2") {
+    runTestAssert[Stat](
+      """|{
+         |  freezing
+         |
+         |    | !
+         |}
+         |""".stripMargin,
+      """|{
+         |  freezing
+         |  `|`!
+         |}
+         |""".stripMargin
+    )(blk("freezing", tpostfix("|", "!")))
+  }
+
+  test("scala3 infix syntax 3.2.3") {
+    runTestAssert[Stat](
+      """|{
+         |  freezing
+         |
+         |    `|` !
+         |}
+         |""".stripMargin,
+      """|{
+         |  freezing
+         |  `|`!
+         |}
+         |""".stripMargin
+    )(blk("freezing", tpostfix("|", "!")))
+  }
+
+  test("scala3 infix syntax 3.2.4") {
+    runTestAssert[Stat](
+      """|{
+         |  freezing
+         |
+         |    | |
+         |}
+         |""".stripMargin,
+      """|{
+         |  freezing
+         |  `|`|
+         |}
+         |""".stripMargin
+    )(blk("freezing", tpostfix("|", "|")))
+  }
+
+  test("scala3 infix syntax 3.2.5") {
```

Comment:
Do we need to test 3.2.x ? People should be using at least LTS 3.3.x version, so it's highly unexpected for them to be using 3.2.x

Reply:
this is not a scala version but a numbered list of test cases.

there were several examples shown on the documentation page, and i created tests for each (1, 2, 3). then i added variations and numbered them 3.1 and 3.2, for instance.

here, i am adding variations of variations, hence 3.2.1 etc.

Reply:
Aaa that makes sense, I could not figure out why this changed in 3.2.x line :sweat_smile: